src/transformers/training_args.py

```diff
@@ -113,6 +113,7 @@ class OptimizerNames(ExplicitEnum):
     ADAMW_APEX_FUSED = "adamw_apex_fused"
     ADAFACTOR = "adafactor"
     ADAMW_BNB = "adamw_bnb_8bit"
+    ANYPRECISION_ADAMW = "anyprecision_adamw"
```

Comment:
Perhaps let's just call it "adamw_anyprecision" to match all the other `/^adam_/ optims?

Reply:
This does not seem to have been addressed.